src/main/kotlin/oop/Adapter/USRobotsAdaptee.kt

```diff
@@ -0,0 +1,19 @@
+package oop.Adapter
+
+interface USRobotsAdaptee{
+  var speedInMilesPerHour: Double
+  fun enableFirstLawMode()
+  fun jump(feet: Double)
+}
+
+class USRobot(override var speedInMilesPerHour: Double = 0.0) :USRobotsAdaptee{
```

Comment:
I can see why it's better to be a data class, but if I make ```speedInMilesPerHour``` val, later in the setter of the adapter, converting kilometers to miles, I can't set a new value since it's immutable. Should I change also the adapter?